service_contracts.rst

```diff
@@ -338,3 +338,11 @@ To handle the rewards, the MonitoringService contract provides two functions. ``
 
 .. autosolcontract:: MonitoringService
     :members: monitor, claimReward
+
+
+monitor()
+---------
+
+When the :ref:`MS` calls ``monitor()`` with a :term:`balance proof`, ``MonitoringService`` smart contract submits the balance proof on behalf of the :ref:`MS`.  If that succeeds, ``MonitoringSergvice`` smart contract records the fact that the :ref:`MS` has submitted the balance proof.
+
+* ``monitor()`` refuses callers that are not registered in ServiceRegistry.
```

Comment:
Should `ServiceRegistry` be code styled as well?

Reply:
I fixed this as well.